src/fusiontrees/manipulations.jl

```diff
@@ -766,7 +768,32 @@ function artin_braid(f::FusionTree{I, N}, i; inv::Bool = false) where {I<:Sector
     inner = f.innerlines
     vertices = f.vertices
     u = one(I)
+
+    if u in (uncoupled[i],uncoupled[i+1]) # the braid simplifies drastically, we are braiding a trivial sector
+        a, b = uncoupled[i], uncoupled[i+1]
+        uncoupled′ = TupleTools.setindex(uncoupled,b,i);
+        uncoupled′ = TupleTools.setindex(uncoupled′,a,i+1);
+        vertices′ = vertices;
+
+        if i > 1 #we also need to alter innerlines and vertices
+            incharges = (uncoupled[1],inner...,coupled′);
+
+            vertices′  = TupleTools.setindex(vertices′,vertices[i],i-1)
+            vertices′  = TupleTools.setindex(vertices′,vertices[i-1],i)
+
+            if a == u
+                inner = TupleTools.setindex(inner,incharges[i+1],i-1);
+            else
+                inner = TupleTools.setindex(inner,incharges[i-1],i-1);
+            end
+        end
+
+        f′ = FusionTree{I}(uncoupled′, coupled′, isdual′, inner, vertices′)
+        return fusiontreedict(I)(f′ => true)
+    end
+
     oneT = one(eltype(Rsymbol(u,u,u))) * one(eltype(Fsymbol(u,u,u,u,u,u)))
+
```

Comment:
Move the line of `oneT` to above your `if` clause, and make line 792 into
```julia
fusiontreedict(I)(f′ => oneT)
```

Reply:
but then Rsymbol(u,u,u) needs to be defined for trivial irreps

Reply:
Ah no, that's the point, it does not exist for NoBraiding()

Reply:
Just make it 
```julia
if BraidingStyle(I) isa NoBraiding
oneT = one(eltype(Fsymbol(u,u,u,u,u,u)))
else
oneT = one(eltype(Rsymbol(u,u,u))) * one(eltype(Fsymbol(u,u,u,u,u,u)))
end
```
above your `if`. Then 
```julia
fusiontreedict(I)(f′ => oneT)
```
and below your `if` clause maybe add
```julia
BraidingStyle(I) isa NoBraiding && throw(SectorMismatch("cannot braid sector "*type_repr(I)))
```
so that the compiler knows that it can never go below that line